msteams-platform/concepts/deploy-and-publish/appsource/prepare/submission-checklist.md

```diff
@@ -1,90 +1,53 @@
+1. If present, [fix issues with your manifest](~/resources/schema/manifest-schema.md) or app in general.
 
-### Teams app validator
+## Provide notes for testing your app
 
-The **Validate** page allows you to check your app package before submission to AppSource. Simply upload your app package and the validation tool will check your app against all manifest-related test cases. For each failed test, the description provides a documentation link to help you fix the error.
+Include the following information with your submission. If you upload test notes to SharePoint, you must provide a public link.
 
-![Validation tool](../../../../assets/images/validation-tool/validator.png)
+### Feature list
 
-### Preliminary checklist
+Detail all of the capabilities the app offers within Teams and steps for testing each feature.
 
-For test scenarios that are difficult to automate, the preliminary checklist surfaces seven of the most commonly failed test cases.
+### Accounts
 
-![Preliminary checklist](../../../../assets/images/validation-tool/preliminary-checklist.png)
+* Credentials for at least two account (one admin and one non-admin). For verification purposes, the accounts should have sufficient pre-populated data.
+* A test account is required if your app only allows licensed accounts or safelisting from the backend. Also, if your app has a team or group chat scope, you must provide two test accounts in the same tenant to validate the collaboration scenario.
+* For apps that require a subscription, have an Microsoft 365 tenant/domain dependency, or are for enterprise users, you must provide a third account in the same domain that isn't pre-configured for your app so that we can validate the first-run user experience.
+* If your app has premium features, an account with the necessary access must be provided to test that experience.
 
-## Privacy policy, terms of use and support URLs
+### Integration steps
 
-### Privacy policy
+If a Teams tenant requires configuration to use the app, include the configuration steps and admin and non-admin accounts for validation.
 
-Privacy policy guidelines:
+You can sign up for a [Microsoft 365 Developer Program](https://developer.microsoft.com/microsoft-365/dev-program) subscription. It's *free* for 90 days and renews as long as you're using it for development activities.
 
-> [!div class="checklist"]
->
-> * The privacy policy can be specific to your app and/or an overall policy for all of your services.
-> * If you use a generic privacy policy, it must reference "services", "applications", and "platforms" to include your Teams app as well as your website.
-> * It must include how you handle user data storage, user data retention, deletion, and security controls.
-> * It must include your contact information.
-> * It should not contain broken links, beta URLs, or staging URLs.
+### Video (optional)
 
-### Terms of use
+You can provide a recording of the product so we can fully understand its functionality.
 
```

Comment:
@heath-hamilton include - 
**Create a Partner Center account**
To publish your app to the Teams store and AppSource, you must first set up a developer account - https://docs.microsoft.com/en-us/office/dev/store/open-a-developer-account


**App category mapping**
During submission, you're asked to categorize your app. The following table maps Teams store categories to the categories listed in Partner Center.

Teams categories | Partner Center categories
-- | --
Analytics and BI | Analytics, Data Visualization and BI
Developer and IT | Developer Tools, IT Admin
Education | Education
Human resources | Human Resources and Recruiting
Productivity | Content Management, Files and documents, Productivity, Training and Tutorials, and Utilities
Project management | Communication, Project Management, Workflow, and Business Management
Sales and support | Customer and Contact Management, Customer Support, Financial Management, Sales and Marketing
Social and fun | Image and Video Galleries, Lifestyle, News and Weather, Social, Travel, and Navigation

Reply:
@ramendhe Just checking if we should instead refer to the M365 store submission FAQs so information is always relevant? https://docs.microsoft.com/en-us/office/dev/store/appsource-submission-faq